irmaclient/client.go

```diff
@@ -80,10 +80,17 @@ type KeyshareHandler interface {
 	EnrollmentSuccess(manager irma.SchemeManagerIdentifier)
 }
 
+type ChangepinHandler interface {
+	ChangepinFailure(manager irma.SchemeManagerIdentifier, err error)
+	ChangepinSuccess(manager irma.SchemeManagerIdentifier)
+	ChangepinIncorrect(manager irma.SchemeManagerIdentifier)
+}
```

Comment:
PIN is a separate word from Change, but they are concatenated here, so `ChangePin` would be preferable - here and all other places where `pin` is concatenated with some other word (including`irma_mobile` and `irma_keyshare_server`).